README.md

```diff
@@ -225,46 +265,63 @@ file inside of `<build_dir>`.
 meson setup <build_dir>
 ```
 
-#### ... with `intercept-build`
+#### ... with `bear`
 
-`intercept-build` (part of the [scan-build tool](https://github.com/rizsotto/scan-build))
-is recommended for non-`cmake` projects.
-`intercept-build` is bundled with `clang` under `tools/scan-build-py`,
-but a standalone version can be easily installed via `pip` with:
+[`bear`](https://github.com/rizsotto/Bear) is recommended for projects whose build systems
+don't generated `compile_commands.json` automatically
```

Comment:
```suggestion
don't generate `compile_commands.json` automatically
```

Reply:
Fixed in [`f311083..87eebdf`](https://github.com/immunant/c2rust/compare/f31108342256aa1aba8301732ba2bdc40f170eec..87eebdf80c9194ce25f21e05bd308c6c59639ed4).